app/frontend/src/App.js

```diff
@@ -21,6 +22,7 @@ function App() {
     <div className="App">
       <BrowserRouter>
         <Navibar cyClass="cy-navBarNav"/>
+        <ErrorModal />
```

Comment:
Can you please explain to me how this works? Because ErrorModal is never opened, is this like the break tag where you only have one that has /> in it